Sources/FilterViewController/FilterViewController.swift

```diff
@@ -4,161 +4,77 @@
 
 import Foundation
 
-public protocol FilterViewControllerDelegate: AnyObject {
-    func applyFilterButtonTapped()
+public protocol ApplySelectionButtonOwner: class {
+    func showApplyButton(_ show: Bool, animated: Bool)
```

Comment:
Hmm, I'm not sure. In my mind, the child doesn't always show the button when its parent does, but the parent will always show it when the child does. This might be different than what we want however.

Reply:
If you are in a multilevel list and you have changed something so the apply button is visible it should still be visible when you open a new child...

Reply:
Yep, I agree! Fixing it ;)